lighthouse-core/audits/first-meaningful-paint.js

```diff
@@ -133,9 +136,15 @@ class FirstMeaningfulPaint extends Audit {
     // Our navStart will be the latest one before fCP.
     const navigationStart = frameEvents.filter(e =>
         e.name === 'navigationStart' && e.ts < firstFCP.ts).pop();
-    // FMP will follow at/after the FCP
+    // fMP will follow at/after the FCP, though we allow some timestamp tolerance
     const firstMeaningfulPaint = frameEvents.find(e =>
-        e.name === 'firstMeaningfulPaint' && e.ts >= firstFCP.ts);
+        e.name === 'firstMeaningfulPaint' && e.ts >= (firstFCP.ts - fCPfMP_TOLERANCE));
```

Comment:
if there's an FMP event a few ms before FCP (within tolerance) *and* one after FCP, is the first one always better here?